docs/customising/make_a_new_theme.md

```diff
@@ -79,7 +79,16 @@ setting in the config file.
 Set it to the URL of your uniquely named repo, for example:
 
     THEME_URLS:
-         - 'git://github.com/yourgithub/abcexample-theme.git'
+         - 'https://github.com/yourgithub/abcexample-theme.git'
+
+
+<div class="attention-box helpful-hint">
+  <p>
+    If you're familiar with git and already have ssh set up, feel free to use
+    the ssh format - <code>ssh://yourname@git.github.com/yourgithub/abcexample-theme.git</code>
```

Comment:
The SSH URL given by GitHub is like `git@github.com:mysociety/alavetelitheme.git`:

![screen shot 2016-04-01 at 15 01 33](https://cloud.githubusercontent.com/assets/282788/14209152/a5168e8e-f81a-11e5-8884-4084044a7ba7.png)

Haven't tested the form above, but just want to confirm its okay?

Might be worth linking "the ssh format" to https://help.github.com/articles/which-remote-url-should-i-use/?